standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -3214,15 +3290,51 @@ private List<Path> dropPartitionsAndGetLocations(RawStore ms, String catName, St
   }
 
   @Override
+  @Deprecated
   public void drop_table(final String dbname, final String name, final boolean deleteData)
       throws NoSuchObjectException, MetaException {
-    drop_table_with_environment_context(dbname, name, deleteData, null);
+    startFunction("drop_table", ": " + dbname + ":" + name);
+    boolean success = false;
+    Exception ex = null;
+    try {
+    String[] parsedDbName = parseDbName(dbname, conf);
```

Comment:
can we just call `drop_table_core` in these old deprecated methods or just leave as it is?

Reply:
The idea is to point the deprecated methods to new methods so that functionality remains unaffected when we remove these methods in the next major release